powerflex_data/environment.yml

```diff
@@ -0,0 +1,52 @@
+name: powerflex_data_env
+channels:
+  - defaults
+dependencies:
+  - asn1crypto=1.3.0=py38_0
+  - blas=1.0=mkl
+  - boto3=1.12.12=py_0
+  - botocore=1.15.12=py_0
+  - ca-certificates=2020.1.1=0
+  - certifi=2019.11.28=py38_0
+  - cffi=1.14.0=py38hb5b8e2f_0
+  - chardet=3.0.4=py38_1003
+  - cryptography=2.8=py38ha12b0ac_0
+  - docutils=0.15.2=py38_0
+  - fsspec=0.6.2=py_0
+  - idna=2.8=py38_1000
+  - intel-openmp=2019.4=233
+  - jmespath=0.9.4=py_0
+  - libcxx=4.0.1=hcfea43d_1
+  - libcxxabi=4.0.1=hcfea43d_1
+  - libedit=3.1.20181209=hb402a30_0
+  - libffi=3.2.1=h475c297_4
+  - libgfortran=3.0.1=h93005f0_2
+  - mkl=2019.4=233
+  - mkl-service=2.3.0=py38hfbe908c_0
+  - mkl_fft=1.0.15=py38h5e564d8_0
+  - mkl_random=1.1.0=py38h6440ff4_0
+  - ncurses=6.2=h0a44026_0
+  - numpy=1.18.1=py38h7241aed_0
+  - numpy-base=1.18.1=py38h6575580_1
+  - openssl=1.1.1d=h1de35cc_4
+  - pandas=1.0.1=py38h6c726b0_0
+  - pip=20.0.2=py38_1
+  - pycparser=2.19=py_0
+  - pyopenssl=19.1.0=py38_0
+  - pysocks=1.7.1=py38_0
+  - python=3.8.1=h359304d_1
+  - python-dateutil=2.8.1=py_0
+  - pytz=2019.3=py_0
+  - readline=7.0=h1de35cc_5
+  - requests=2.22.0=py38_1
+  - s3fs=0.4.0=py_0
+  - s3transfer=0.3.3=py38_0
+  - setuptools=45.2.0=py38_0
+  - six=1.14.0=py38_0
+  - sqlite=3.31.1=ha441bb4_0
+  - tk=8.6.8=ha441bb4_0
+  - urllib3=1.25.8=py38_0
+  - wheel=0.34.2=py38_0
+  - xz=5.2.4=h1de35cc_4
+  - zlib=1.2.11=h1de35cc_3
+prefix: /Users/derins/miniconda3/envs/powerflex_data_env
```

Comment:
This yml needs revamped. Please remove your personal path from the prefix. And remove all dependencies listed other than the ones required by the project. Even though conda adds additional ones. Don't list them here for Devine.

Reply:
Unused dependencies and prefix is removed.